### DISCUSSION TITLE
Welcome to meroshare-bot Discussions!

### DISCUSSION THREAD

**Discussion by bipin0x01** [bipin0x01/meroshare-bot#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by krishna-ji** [comment]
i found this script today. but the script is stuck on the process. after displaying "  number of account detected 7." what am i doing wrong?
